tests/virt_autotest/prepare_non_transactional_server.pm

```diff
@@ -0,0 +1,100 @@
+# PREPARE NON-TRANSACTIONAL SERVER MODULE
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: This module do preparation work for virtual machine installation on non
+# -transactional server, including extensions registration, packages installation,
+# services toggle and other operations being pertinent to non-transactonal server
+# or which need to to performed in advance for various purposes, including better
+# test flow, clear functional division, placeholder for future extension and etc.
+#
+# Maintainer: Wayne Chen <wchen@suse.com> qe-virt@suse.de
+package prepare_non_transactional_server;
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use transactional;
+use utils;
+use version_utils;
+use Utils::Systemd;
+use Utils::Backends qw(get_serial_console);
+use ipmi_backend_utils;
+use virt_autotest::utils;
+
+sub run {
+    my $self = shift;
+
+    $self->prepare_ground;
+    $self->prepare_extensions;
+    $self->prepare_packages;
+    $self->prepare_bootloader;
+    $self->prepare_services;
+    $self->prepare_reboot;
+    $self->restore_ground;
+}
+
+sub prepare_ground {
+    my $self = shift;
+
+    set_var('_NEEDS_REBOOTING', get_var('NEEDS_REBOOTING', 0));
+    set_var('NEEDS_REBOOTING', 0);
+}
+
+sub prepare_extensions {
+    my $self = shift;
+
+    zypper_call("install --no-allow-downgrade --no-allow-name-change --no-allow-vendor-change suseconnect-ng");
+    virt_autotest::utils::subscribe_extensions_and_modules;
+}
+
+sub prepare_packages {
+    my $self = shift;
+
+    my $zypper_install_package = "install --no-allow-downgrade --no-allow-name-change --no-allow-vendor-change libspice-server1 qemu-audio-spice qemu-chardev-spice qemu-spice qemu-ui-spice-core spice-vdagent";
+    zypper_call("$zypper_install_package");
+    install_extra_packages;
```

Comment:
It will be also good to explain pkgs in INSTALL_OTHER_PACKAGES too, if possible.